core/modules/ccontrol/UserQuerySelect.cc

```diff
@@ -146,11 +146,12 @@ UserQuerySelect::UserQuerySelect(std::shared_ptr<qproc::QuerySession> const& qs,
                                  std::shared_ptr<qproc::SecondaryIndex> const& secondaryIndex,
                                  std::shared_ptr<qmeta::QMeta> const& queryMetadata,
                                  qmeta::CzarId czarId,
+                                 uint64_t taskId,
                                  std::string const& errorExtra)
     :  _qSession(qs), _messageStore(messageStore), _executive(executive),
        _infileMergerConfig(infileMergerConfig), _secondaryIndex(secondaryIndex),
        _queryMetadata(queryMetadata), _qMetaCzarId(czarId), _qMetaQueryId(0),
-       _killed(false), _submitted(false), _sessionId(0), _sequence(0),
+       _killed(false), _submitted(false), _taskId(taskId), _sequence(0),
```

Comment:
yes, userQueryId / chunkQueryId type names would be very clear. ("task" always was a little less so to me)